frontend/src/Table/InequalityFilter.tsx

```diff
@@ -0,0 +1,123 @@
+    const [currentOperator, currentValue = null] = current || [];
+
+    const [inputValue, setInputValue] = useState<number | null>(currentValue);
+    const [inputOperator, setInputOperator] = useState<InequalityOperator>(
+        currentOperator || "EQ"
+    );
+
+    useEffect(() => {
+        const didValueChange = currentValue !== inputValue;
+        const didOperatorChange = currentOperator !== inputOperator;
+
+        if (didValueChange) {
+            onChange(
+                isDefined(inputValue) ? [inputOperator, inputValue] : null
+            );
+        } else if (didOperatorChange && isDefined(inputValue)) {
+            onChange([inputOperator, inputValue]);
+        }
+    }, [currentOperator, currentValue, inputOperator, inputValue]);
+
+    const commonStyle = {
+        background: "white",
+        fontSize: "inherit",
+        height: TABLE_FILTER_HEIGHT,
+        maxHeight: "100%",
+        minWidth: 0,
+        minHeight: 0,
+        lineHeight: 0,
+    };
+
+    return (
+        <Box
+            display="flex"
+            alignItems="end"
+            justifyContent="center"
+            height="100%"
+            maxWidth={width}
+        >
+            <Select
+                size="sm"
+                value={inputOperator}
+                onChange={(_, o) => setInputOperator(o || "EQ")}
+                sx={{
+                    ...commonStyle,
+                    width: "fit-content",
+                    pl: "4px",
+                    mr: "1px",
+                }}
+            >
+                {(["EQ", "GT", "LT"] as const).map((o) => (
+                    <Option key={o} value={o}>
+                        {operatorLabel(o)}
+                    </Option>
+                ))}
+            </Select>
+
+            <Select
+                size="sm"
+                placeholder={placeholder}
+                value={inputValue}
+                onChange={(_, value) => setInputValue(value)}
+                indicator={isDefined(inputValue) ? null : undefined}
+                sx={{ ...commonStyle, flex: 1 }}
+                endDecorator={
+                    isDefined(inputValue) ? (
+                        <ResetButton reset={() => setInputValue(null)} />
+                    ) : undefined
+                }
+            >
+                {range(min, max + 1).map((value) => (
+                    <Option key={value} value={value}>
+                        {value}
+                    </Option>
+                ))}
+            </Select>
+        </Box>
+    );
+}
+
+function ResetButton(props: { reset: () => void }) {
+    const { reset } = props;
+
+    return (
+        <IconButton
+            size="sm"
+            onMouseDown={(e) => e.stopPropagation()}
+            onClick={reset}
+            sx={{ minWidth: 0, minHeight: 0, lineHeight: 0, p: "1px" }}
+        >
+            <ClearIcon />
+        </IconButton>
+    );
+}
+
+function operatorLabel(operator: InequalityOperator) {
```

Comment:
Possible enhancement: we could provide `>=` and `<=` inequalities on the FE, converting them into a filter expression. For example the `>= x` FE label converts to the `GT` and `EQ` operators under the hood.

Reply:
Ah, or at least we could do that, if the API supported it, which it doesn't. Nevermind! If someone wants `>= x` they can just write `> x - 1`. Maths.

Reply:
I like this and we can support it without changing the API! Can just have the FE do the `>= x` to `> x - 1` translation. Shouldn't be too much work 💯

Reply:
Me, a backend engineer.

<img width="1544" height="938" alt="image" src="https://github.com/user-attachments/assets/99944971-2dfc-4f71-a557-ab6140667cbc" />

Reply:
😂 💀